src/modules/fancyzones/FancyZonesLib/FancyZones.cpp

```diff
@@ -1002,7 +1002,6 @@ bool FancyZones::OnSnapHotkey(DWORD vkCode) noexcept
     {
         return (vkCode == VK_LEFT || vkCode == VK_RIGHT) && OnSnapHotkeyBasedOnZoneNumber(window, vkCode);
     }
-    return false;
```

Comment:
I think else statement also could be removed, and instead of `return false`  would be `return (vkCode == VK_LEFT || vkCode == VK_RIGHT) && OnSnapHotkeyBasedOnZoneNumber(window, vkCode);`

Reply:
Good, it makes it cleaner